src/assets/sass/protocol/base/elements/_sections.scss

```diff
@@ -23,25 +23,18 @@
     }
 }
 
-.mzp-s-content {
-    @include border-box;
-    @include clearfix;
-    margin: 0 auto;
+.mzp-t-narrow {
     max-width: $content-max;
     min-width: $content-xs;
-    padding: $layout-xs;
-    position: relative;
 
     @media #{$mq-md} {
         max-width: $content-md;
         min-width: $content-xs;
```

Comment:
We don't need to declare the min-width at each break point. They each inherit the definition from the last. Cascade FTW.